### PR TITLE
[Enhancement] Make Quiet Behaviour Default

### PR DESCRIPTION
**Pull request by kevinmatthes**
This Pull Request is my suggestion in order to implement the feature requested in #14.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,6 +45,7 @@ main directory of this repository.
 | `--message-starts-with`   | `-l`  | (**OR**) Filter for certain commit messages.                              |
 | `--output`                | `-o`  | An output file for the commits per day in CSV format.                     |
 | `--quiet`                 | `-q`  | Hide the output of commit messages.                                       |
+| `--verbose`               | `-V`  | Always show the entire output.                                            |
```
Comment (kevinmatthes) [inline README.md:48]: The new option is named `-V` in its abbreviated form.  Please note the majuscule.

Some projects use rather `-v` (minuscule) therefore, including Git, others apply this convention in order to reserve the minuscule for the version request (which is about to be introduced by another Pull Request in the near future).

Do you prefer the Git way, also regarding the verbosity levels? :wrench:

Comment (wert007) [reply README.md:48]: Hmm, I mean is the version number important enough for a shorthand flag? I think a `-v` for verbose might be more fitting. At least, as long as `-v` does not mean anything else.

File: src/main.rs [hunk 2]
```diff
@@ -3,6 +3,7 @@ use std::{collections::HashMap, error::Error, fs, io::Write, num::ParseIntError,
 fn main() -> Result<(), Box<dyn Error>> {
     let mut opts = getopts::Options::new();
     let opts = opts
+        .optflag("V", "verbose", "Always show the entire output.")
         .optflag("q", "quiet", "Hide the output of commit messages.")
```
Comment (wert007) [inline src/main.rs:7]: Since the quiet flag is not used anymore, it should be removed I think.

**Comment by wert007** [comment]
Also general note. While you really improved your commit messages, I think those two commits could easily have been one. 😄

**Comment by kevinmatthes** [comment]
I decided to document the new feature first in order to avoid forgetting about it.  When I then turned to the implementation, I was surprised how small the change actually was.  This is why the suggestion has those two initial commits.

**Comment by wert007** [comment]
Fair enough. I think you can squash them with git rebase.

**Comment by kevinmatthes** [comment]
In contrast to the previous Pull Requests, I gave the changes a branch on their own and avoided committing directly into `main`.  Hence, I expect GitHub to squash them automatically when choosing the corresponding option during the merge.  But this is just an assumption which I do not have any examples for.

File: README.md (hunk 1 above)
Comment (kevinmatthes) [reply README.md]: :wrench: :heavy_check_mark:

File: src/main.rs (hunk 2 above)
Comment (kevinmatthes) [reply src/main.rs]: :wrench: :heavy_check_mark:

**Comment by wert007** [comment]
Well, this is a learning experience for the both of us 😄

Looks good to merge for me, if that's fine with you

**Comment by kevinmatthes** [comment]
Yes, thanks. :+1:

**Pull request merged**